gio/src/subprocess_launcher.rs

```diff
@@ -1,7 +1,7 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
 #[cfg(any(unix, all(docsrs, unix)))]
-use std::os::unix::io::IntoRawFd;
+use std::os::unix::io::{IntoRawFd, OwnedFd};
```

Comment:
Below: "unused import: `ffi`" on Windows so maybe just move that part into a `#[cfg(unix)]` while we're at it